subprojects/architecture-test/src/test/java/org/gradle/architecture/test/InternalNullabilityTest.java

```diff
@@ -0,0 +1,664 @@
+        "org.gradle.api.publish.ivy.internal",
+        "org.gradle.api.publish.ivy.internal.artifact",
+        "org.gradle.api.publish.ivy.internal.dependency",
+        "org.gradle.api.publish.ivy.internal.publication",
+        "org.gradle.api.publish.ivy.internal.publisher",
+        "org.gradle.api.publish.ivy.internal.versionmapping",
+        "org.gradle.api.publish.ivy.plugins",
+        "org.gradle.api.publish.ivy.tasks",
+        "org.gradle.api.publish.maven.internal",
+        "org.gradle.api.publish.maven.internal.artifact",
+        "org.gradle.api.publish.maven.internal.dependencies",
+        "org.gradle.api.publish.maven.internal.publication",
+        "org.gradle.api.publish.maven.internal.publisher",
+        "org.gradle.api.publish.maven.internal.tasks",
+        "org.gradle.api.publish.maven.internal.validation",
+        "org.gradle.api.publish.maven.plugins",
+        "org.gradle.api.publish.maven.tasks",
+        "org.gradle.api.publish.plugins",
+        "org.gradle.api.publish.tasks",
+        "org.gradle.api.reflect",
+        "org.gradle.api.reporting.components.internal",
+        "org.gradle.api.reporting.model.internal",
+        "org.gradle.api.reporting.plugins",
+        "org.gradle.api.resources.internal",
+        "org.gradle.api.specs",
+        "org.gradle.api.specs.internal",
+        "org.gradle.api.tasks.bundling.internal",
+        "org.gradle.api.tasks.diagnostics.internal.dependencies",
+        "org.gradle.api.tasks.diagnostics.internal.dsl",
+        "org.gradle.api.tasks.diagnostics.internal.graph",
+        "org.gradle.api.tasks.diagnostics.internal.insight",
+        "org.gradle.api.tasks.diagnostics.internal.text",
+        "org.gradle.api.tasks.internal",
+        "org.gradle.api.tasks.scala.internal",
+        "org.gradle.api.tasks.testing",
+        "org.gradle.api.tasks.testing.testng",
+        "org.gradle.api.tasks.util.internal",
+        "org.gradle.api.tasks.wrapper",
+        "org.gradle.architecture.library.freeze",
+        "org.gradle.architecture.test",
+        "org.gradle.buildinit.plugins",
+        "org.gradle.buildinit.plugins.internal",
+        "org.gradle.buildinit.plugins.internal.action",
+        "org.gradle.buildinit.plugins.internal.maven",
+        "org.gradle.buildinit.plugins.internal.model",
+        "org.gradle.buildinit.plugins.internal.modifiers",
+        "org.gradle.buildinit.plugins.internal.services",
+        "org.gradle.cache.internal.btree",
+        "org.gradle.cache.internal.cacheops",
+        "org.gradle.cache.internal.filelock",
+        "org.gradle.cache.internal.locklistener",
+        "org.gradle.cache.internal.scopes",
+        "org.gradle.cache.internal.streams",
+        "org.gradle.cache.scopes",
+        "org.gradle.cli",
+        "org.gradle.composite.internal",
+        "org.gradle.composite.internal.plugins",
+        "org.gradle.configuration",
+        "org.gradle.configuration.project",
+        "org.gradle.configurationcache",
```

Comment:
:question: Shouldn't the `org.gradle.configurationcache.*` packages be excluded as written in Kotlin?

Reply:
Thank you for noticing this. Either the predicate does not work as expected, or I wired it incorrectly.
https://github.com/gradle/gradle/pull/24579/files#diff-3a56d557a8a596ae8247924d791beb58dc0fd1fa24d5fa08a85ad3f16ee175f3R78

Reply:
Okay, this one is solved. I was collecting the pending list before excluding the Kotlin stuff. Removed the CC package from pending, and the test still passes ✅